authentication-typescript/template/src/actions/RegistrationUIActions.tsx

```diff
@@ -30,35 +30,58 @@ export const ProjectRegistrationUIActions: () => RegistrationUIActions = () => (
      *
      * @returns Resolve with EULA, otherwise reject with an error message.
      */
-    loadEULA: async (language: string): Promise<string> => {
+    loadEula: async (language: string): Promise<string> => {
         await sleep(1000);
 
         if (isRandomFailure()) {
             throw new Error('Sorry, there was a problem sending your request.');
         }
 
-        if (language !== 'en' && language !== 'en_US') {
+        if (!language.includes('en')) {
             return 'Other language EULA';
         }
 
         return SAMPLE_EULA;
     },
 
     /**
-     * The user entered their email address and accepted the EULA.
-     * The API should now send them an email with the validation code.
+     * The user accepted the EULA.
+     * The API should now update accepted EULA.
      *
-     * @param email  Email for the registering user.
      *
      * @returns Resolve when the server accepted the request.
      */
-    requestRegistrationCode: async (email: string): Promise<void> => {
+    acceptEula: async (): Promise<void> => {
         await sleep(800);
         if (isRandomFailure()) {
             throw new Error('Sorry, there was a problem sending your request.');
         }
     },
 
+    requestRegistrationCode: async (email: string): Promise<string> => {
```

Comment:
These functions still need comments/descriptions — every function in this file should have one